### PR TITLE
Update (almost) all our dependencies.

### PR DESCRIPTION
**Pull request by pmuetschard**
Updates all our 3rd party dependencies except for breakpad, llvm, and SWT & JFace.

### CODE REVIEW

**Comment by chrisforbes** [comment]
Some widget sizes are broken with the switch to GTK3, see ends of checkbox labels are clipped off:
Screenshot from 2020-03-24 15-57-28

**Comment by chrisforbes** [comment]
Vertical alignment of "Regex" checkboxes with filter textboxes is broken with switch to GTK3:
Screenshot from 2020-03-24 16-01-13

**Comment by pmuetschard** [comment]
I've removed the SWT & JFace update commits from the PR, since they have a bit too much effect.

File: core/fault/stacktrace/stacktrace_test.go [hunk 1]
```diff
@@ -82,7 +82,7 @@ var (
 			{
 				"⇒ core/fault/stacktrace/stacktrace_test.go@36:nested1",
 				"⇒ core/fault/stacktrace/stacktrace_test.go@34:nested3",
-				"⇒ core/fault/stacktrace/stacktrace_test.go@34:nested3",
+				"⇒ core/fault/stacktrace/stacktrace_test.go@36:nested1",
```
Comment (chrisforbes) [inline core/fault/stacktrace/stacktrace_test.go]: This seems dubious?

File: gapic/src/platform/linux/com/google/gapid/glcanvas/GlCanvas.java [hunk 2]
```diff
@@ -48,7 +48,7 @@
   static {
     boolean loaded = false;
     try {
-      Library.loadSystem("linux_glcanvas");
+      Library.loadSystem("", "linux_glcanvas");
```
Comment (hevrard) [inline gapic/src/platform/linux/com/google/gapid/glcanvas/GlCanvas.java]: I don't understand that change? The org.lwjgl.system.Library documentation does not seem to have a loadSystem() function that could take two strings?

https://javadoc.lwjgl.org/org/lwjgl/system/Library.html

Comment (pmuetschard) [reply gapic/src/platform/linux/com/google/gapid/glcanvas/GlCanvas.java]: https://github.com/LWJGL/lwjgl3/blob/master/modules/lwjgl/core/src/main/java/org/lwjgl/system/Library.java#L61

**Comment by pmuetschard** [comment]
> Also, would it be possible to update grpc to be >= 1.24

No, please see the comment in the commit.

File: core/fault/stacktrace/stacktrace_test.go (hunk 1 above)
Comment (pmuetschard) [reply core/fault/stacktrace/stacktrace_test.go]: Done. Figured out how to make it stable.

**Pull request merged**